src/ai/backend/common/logging.py

```diff
@@ -88,6 +88,8 @@
         t.Key("ca-certs", default=None): t.Null | t.String(allow_blank=True),
         t.Key("keyfile", default=None): t.Null | t.String(allow_blank=True),
         t.Key("certfile", default=None): t.Null | t.String(allow_blank=True),
+        t.Key("fqdn", default=True): t.Bool,
+        t.Key("localname", default=None): t.Null | t.String(allow_blank=True),
```

Comment:
Allowing the use of a blank `localname` does not appear to be secure, following the implementation details in the `graypy`:

https://github.com/severb/graypy/blob/a5a82c535002db6015473cb59a7476361060d358/graypy/handler.py#L236-L237